src/System.Linq.Expressions/src/System/Linq/Expressions/ExpressionStringBuilder.cs

```diff
@@ -717,21 +658,31 @@ protected internal override Expression VisitDefault(DefaultExpression node)
         {
             Out("default(");
             Out(node.Type.Name);
-            Out(")");
+            Out(')');
             return node;
         }
 
         protected internal override Expression VisitLabel(LabelExpression node)
         {
             Out("{ ... } ");
             DumpLabel(node.Target);
-            Out(":");
+            Out(':');
             return node;
         }
 
         protected internal override Expression VisitGoto(GotoExpression node)
         {
-            Out(node.Kind.ToString().ToLower());
+            string op;
+            switch (node.Kind)
+            {
+                case GotoExpressionKind.Goto:     op = "goto";     break;
+                case GotoExpressionKind.Break:    op = "break";    break;
+                case GotoExpressionKind.Continue: op = "continue"; break;
+                case GotoExpressionKind.Return:   op = "return";   break;
+                default:
+                    throw new InvalidOperationException();
+            }
+            Out(op);
```

Comment:
Note that no space is emitted here, making the keyword glued to the target label's name. This is addressed in a separate PR (https://github.com/dotnet/corefx/pull/11427) to judge against the breaking change bar.